### PR TITLE
fix: Use a bundled, statically linked libcurl and openssl

### PR DESCRIPTION
**Pull request by kamilogorek**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,19 +1,20 @@
 # Changelog
 
+## sentry-cli 1.51.0
+
+* feat: Add `dist` option to `react-native appcenter` command (#653)
+* ref: Notify user about missing `sudo` command instead of incorrect "No such file or directory" when updating/uninstalling `sentry-cli` (#656)
+* fix: Remove `Closing connection 0` warnings after every HTTP request (#657)
```
Comment (rhcarvalho) [inline CHANGELOG.md]: #657 refers to this very PR, is this fixed by bundling curl?!

Comment (kamilogorek) [reply CHANGELOG.md]: Yup, it has been fixed in `curl 7.65.0`, where OSX is shipped with an older version by default. See: https://github.com/curl/curl/commit/d934059afd8568201d0646d6bfd201f39b6bc412

Comment (rhcarvalho) [reply CHANGELOG.md]: Could we throw a reference to this in the commit message? Would make it easier for anyone later trying to understand where the problem was coming from and how it got solved :)

Comment (kamilogorek) [reply CHANGELOG.md]: Will do once merging 👌

**Comment by kamilogorek** [comment]
@rhcarvalho ref: https://github.com/getsentry/sentry-cli/commit/33f037641209eaa5b3ea75d300d8451f2b2af6ec

**Pull request merged**